### ISSUE TITLE
Add tests

### ISSUE THREAD

**Issue by smukherj1** [bazel-contrib/vscode-bazel#32]
We currently have no tests for the extension. VS Code extension testing framework described here https://code.visualstudio.com/docs/extensions/testing-extensions

We will need to add a starter test and include it in the PR verification CI job

**Comment by allevato** [comment]
Yes, we definitely need some good coverage for our lower-level APIs here, at least. (I'm not sure what VS Code offers in the area of extension-based testing.)

We'll need to tweak some of our components where we invoke Bazel, Buildifier, etc. so that we can inject fakes that mock the results. We should probably wait on that until I make the changes that replace the XML-based processing with the proto output.
